plugins/source/aws/resources/services/accessanalyzer/analyzers.go

```diff
@@ -42,7 +42,6 @@ func fetchAccessanalyzerAnalyzers(ctx context.Context, meta schema.ClientMeta, p
 	svc := c.Services().Accessanalyzer
 	paginator := accessanalyzer.NewListAnalyzersPaginator(svc, &accessanalyzer.ListAnalyzersInput{})
 	for paginator.HasMorePages() {
-		// no need to override API call options anymore: https://github.com/aws/aws-sdk-go-v2/issues/1260
```

Comment:
we used to override the region due to a bug in the sdk, now we do it because our sdk clients aren't per account+region...